homeassistant/components/switch/wemo.py

```diff
@@ -199,9 +201,33 @@ def turn_off(self, **kwargs):
         self.wemo.off()
         self.schedule_update_ha_state()
```

Comment:
This should probably be removed, since we have callbacks for state change called from pywemo.